docs/workshop-schedule.md

```diff
@@ -33,17 +33,17 @@ nav_title: Schedule
 | Time | Topic | Instructor|
 |------|-------|-----------|
 | 9:00 - 9:30 | Pull request basics | Jaclyn |
-| | Slides: [Pull request basics (PDF)](slides/09_pull-requests-basics.pdf) | |
+| | Slides: [Pull request basics (PDF)](slides/08_pull-request-basics.pdf) | |
 | 9:30 - 10:00 | Resolving merge conflicts | Josh |
-| 10:00 - 10:30 | Git workflows | Jaclyn |
-| | Slides: [Git workflows (PDF)](slides/11_git-workflows.pdf) | |
+| 1:00 - 1:45 | Leaving and responding to review | Josh |
```

Comment:
```suggestion
| 10:00 - 10:30 | Leaving and responding to review | Josh |
```

Reply:
I meant to leave this as 45 minutes, which means there were some follow-on effects. But I think it is all correct now.